<launch>
  <param name="cam_1" type="string" value="/usb_cam_1/image_raw"/>
  <param name="cam_2" type="string" value="/usb_cam_2/image_raw"/>
  <param name="img_out" type="string" value="/img_out"/>
  <param name="overlay_1" type="string" value="/overlay_1_out"/>
  <param name="overlay_2" type="string" value="/overlay_2_out"/>
  <node pkg="usb_cam" type="usb_cam_node" name="usb_cam_1" output="screen" required="true">
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node pkg="usb_cam" type="usb_cam_node" name="usb_cam_2" output="screen" required="true">
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node pkg="ros_image_workshop" type="multi_camera_overlay" name="multi_overlay" output="screen" required="true"/>
</launch>
